.env.template

```diff
@@ -3,4 +3,4 @@ FLASK_APP=todo_app/app
 FLASK_ENV=development
 
 # Change the following values for local development.
-SECRET_KEY=secret-key
+#SECRET_KEY=5a22aeec590806b94ef4b123c8a0d8e3
```

Comment:
This doesn't feel right - our template:
A) Doesn't want to contain any secrets!
B) Doesn't want this line commented out unless you've actually stopped the app depending on it